challenge-03/README.md

```diff
@@ -150,8 +175,33 @@ deve conter no retorno da frase acima é "metro" no lugar de "metros".
 método), que será concatenada com a frase de retorno, mostrando a resposta
 correta, de acordo com os dados inseridos no objeto.
 */
-?
+
 
 // Agora, apresente-se ;)
-?
+pessoa.apresentacao = function(){
+
+  var sexo;
+    if( pessoa.sexo === 'Feminino' ){
+      sexo = 'a';
+    } else {
```

Comment:
-1
Para não precisar usar o `else` (que acaba aumentando um pouco a complexidade do código), você pode declarar todas as variáveis que você irá utilizar sempre no início do método, atribuindo à elas valores padrão, e usar somente o `if` para alterar esses valores, se necessário :)